README.md

```diff
@@ -1,57 +1,172 @@
-# Yearn Strategy Mix
+# Yearn Strategy Brownie Mix
 
-### What you'll find here
+## What you'll find here
 
-Solidity Smart Contract template for creating your own contract in the Brownie framework.
+- Basic Solidity Smart Contract for creating your own Yearn Strategy (`contracts/Strategy.sol`)
 
-Interfaces for some of the most used DeFi protocols on ethereum mainnet.
+- Interfaces for some of the most used DeFi protocols on ethereum mainnet. (`interfaces/`)
 
-Sample test suite that runs on mainnet fork.
+- Sample test suite that runs on mainnet fork. (`tests/`)
 
-### How does it work for the User
+This mix is configured for use with [Ganache](https://github.com/trufflesuite/ganache-cli) on a [forked mainnet](https://eth-brownie.readthedocs.io/en/stable/network-management.html#using-a-forked-development-network).
+
+## How does it work for the User
 
 Let's say Alice holds 100 DAI and wants to start earning yield % on them.
 
 For this Alice needs to `DAI.approve(vault.address, 100)`.
 
-Then Alice will call `vault.deposit(100)`.
+Then Alice will call `Vault.deposit(100)`.
 
 Vault will then transfer 100 DAI from Alice to itself, and mint Alice the corresponding shares.
 
-Alice can then redeem those shares using `yVault.withdrawAll()` for the corresponding DAI balance.
+Alice can then redeem those shares using `Vault.withdrawAll()` for the corresponding DAI balance (exchanged at `Vault.pricePerShare()`).
+
+## Installation and Setup
+
+1. [Install Brownie](https://eth-brownie.readthedocs.io/en/stable/install.html) & [Ganache-CLI](https://github.com/trufflesuite/ganache-cli), if you haven't already.
+
+2. Sign up for [Infura](https://infura.io/) and generate an API key. Store it in the `WEB3_INFURA_PROJECT_ID` environment variable.
+
+```bash
+export WEB3_INFURA_PROJECT_ID=YourProjectID
+```
+
+3. Sign up for [Etherscan](www.etherscan.io) and generate an API key. This is required for fetching source codes of the mainnet contracts we will be interacting with. Store the API key in the `ETHERSCAN_TOKEN` environment variable.
+
+```bash
+export ETHERSCAN_TOKEN=YourApiToken
+```
+
+4. Download the mix.
+
+```bash
+brownie bake yearn-strategy
+```
+
+## Basic Use
+
+To deploy the demo Yearn Strategy in a development environment:
+
+1. Open the Brownie console. This automatically launches Ganache on a forked mainnet.
+
+```bash
+$ brownie console
+```
+
+2. Create variables for the Yearn Vault and Want Token addresses. These were obtained from the Yearn Registry. Also, loan the Yearn governance multisig.
 
-### How does it work for the Vault
+```python
+>>> vault = Vault.at("0x9B142C2CDAb89941E9dcd0B6C1cf6dEa378A8D7C")  # yvDAI (v0.2.2)
+>>> token = Token.at("0x6b175474e89094c44da98b954eedeac495271d0f")  # DAI
+>>> gov = "ychad.eth"  # ENS for Yearn Governance Multisig
```

Comment:
Why don't you do `accounts.at("0xychad.eth", force=True)` so you can remove all the accounts[0].

Reply:
because governance is already set for this Vault. `accounts[0]` is the user's account.